dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/property/DssRestChannelProperty.java

```diff
@@ -17,10 +18,11 @@
     private static final long serialVersionUID = -6904414304979344025L;
     private final String host;
     private final int port;
-
+    private final DssLogLevel dssLogLevel;
     @Builder
-    private DssRestChannelProperty(String host, int port) {
+    private DssRestChannelProperty(String host, int port, DssLogLevel dssLogLevel) {
         this.host = host;
         this.port = port;
+        this.dssLogLevel = Objects.nonNull(dssLogLevel) ? dssLogLevel : DssLogLevel.DEBUG;;
```

Comment:
I don't know well. 

``` java
DssRestChannelProperty
                .builder()
                .host(host)
                .port(port)
                .build();
```
Like this. If dssLogLevel is not set, can it be set to dssLogLevel.DEBUG? 

And I think, it seems to need that how to set DssLogLevel.
``` java
// in DssRestServer
public DssRestServer(String host, int port,) {
    this(host, port, DssLogLevel.DEBUG ,false, null);
} 

public DssRestServer(String host, int port, DssLogLevel dssLogLevel) {
    this(host, port, dssLogLevel, false, null);
}
```
and please delete semicolon.